src/actions/authors.js

```diff
@@ -52,8 +52,13 @@ export function fetchAuthors(page=1) {
           const camelizedJson = camelizeKeys(response)
           let items = normalize(camelizedJson.items, arrayOf(authorSchema))
           let meta = camelizedJson.meta
+          let currentPage = meta.page
+          let maxResultsPerPage = meta.maxResults
+          let totalResults = meta.total
```

Comment:
use `_.get(meta, 'page', 1)` to get page field and give the default value as well